packages/cli/src/scripts/bump.ts

```diff
@@ -0,0 +1,11 @@
+import stdout from '../utils/stdout';
+import ControllerFor from '../models/local/ControllerFor';
+import { BumpParams } from './interfaces';
+
+export default async function bumpVersion({ version, packageFile }: BumpParams): Promise<void | never> {
```

Comment:
Question: do the commands use these interfaces like BumpParams?

Reply:
Nope. There might be some shared attributes, though, there are not exact matches.